pkg/compat/metrics.go

```diff
@@ -25,15 +26,38 @@ const (
 )
 
 //
-// Global reporter.
-var Metrics *Reporter
+// Numeric metrics consts
+const (
+	one         = 1
+	nanoToMilli = 1e6
+)
+
+//
+// Global reporters.
+var RequestCountMetrics *Reporter
+var RequestRTTMetrics *Reporter
 
 func init() {
-	Metrics = &Reporter{
+	RequestCountMetrics = &Reporter{
+		counter: promauto.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "mtc_client_count",
+				Help: "MTC client API request counts.",
+			},
+			[]string{
+				Cluster,
+				Component,
+				Function,
+				Kind,
+				Method,
+			}),
+	}
+
+	RequestRTTMetrics = &Reporter{
 		counter: promauto.NewCounterVec(
 			prometheus.CounterOpts{
-				Name: "mtc_client",
-				Help: "MTC client API metrics.",
+				Name: "mtc_client_rtt",
```

Comment:
similarly, I think we need to be more descriptive with what this metric conveys to match convention in openshift metric naming, `mtc_client_request_rtt`